cmd/seed-controller-manager/options.go

```diff
@@ -155,6 +159,9 @@ func newControllerRunOptions() (controllerRunOptions, error) {
 	flag.IntVar(&c.schedulerDefaultReplicas, "scheduler-default-replicas", 1, "The default number of replicas for usercluster schedulers")
 	flag.IntVar(&c.concurrentClusterUpdate, "max-parallel-reconcile", 10, "The default number of resources updates per cluster")
 	flag.IntVar(&c.addonEnforceInterval, "addon-enforce-interval", 5, "Check and ensure default usercluster addons are deployed every interval in minutes. Set to 0 to disable.")
+	flag.IntVar(&c.userClusterLeaderElectLeaseDurationSeconds, "user-cluster-leader-elect-lease-duration", 0, "The lease duration in seconds used by user cluster control plane components having leader election (i.e. controller-manager and scheduler). The default value for the component is used when equal or less than 0.")
+	flag.IntVar(&c.userClusterLeaderElectRenewDeadlineSeconds, "control-plane-leader-elect-renew-deadline", 0, "The lease renew deadline in seconds used by user cluster control plane components having leader election (i.e. controller-manager and scheduler). Should be smaller or equal than control-plane-leader-elect-lease-duration. The default value for the component is used when equal or less than 0.")
+	flag.IntVar(&c.userClusterLeaderElectRetryPeriodSeconds, "control-plane-leader-elect-retry-period", 0, "The duration in seconds that user cluster control plane components having leader election (i.e. controller-manager and scheduler) should wait between attempting acquisition and renewal of a leadership. The default value for the component is used when equal or less than 0.")
```

Comment:
I think that you have forgotten to update the flags. Also, I think we should s/should/must.

```suggestion
	flag.IntVar(&c.userClusterLeaderElectRenewDeadlineSeconds, "user-cluster-leader-elect-renew-deadline", 0, "The lease renew deadline in seconds used by user cluster control plane components having leader election (i.e. controller-manager and scheduler). Must be smaller or equal than user-cluster-leader-elect-lease-duration. The default value for the component is used when equal or less than 0.")
	flag.IntVar(&c.userClusterLeaderElectRetryPeriodSeconds, "user-cluster-leader-elect-retry-period", 0, "The duration in seconds that user cluster control plane components having leader election (i.e. controller-manager and scheduler) should wait between attempting acquisition and renewal of a leadership. The default value for the component is used when equal or less than 0.")
```

Reply:
yeah, no worries. I will remove the flags for the moment to reduce the scope of this PR.